packages/smooth_app/lib/pages/preferences/user_preferences_page.dart

```diff
@@ -165,6 +167,46 @@ class _UserPreferencesPageState extends State<UserPreferencesPage>
               ),
             ),
           ),
+          SliverToBoxAdapter(
```

Comment:
Not sure what the added-value of `SliverToBoxAdapter` is.

Reply:
to centre the login button as per mentioned in the issue #2059 .

Reply:
I mean, the added-value of `SliverToBoxAdapter` compared to simply adding widgets to `children`.

Reply:
Actually in children's it contain list tile and due to list tile it contains margin in leading icon and due to that login button cannot be fitted in center so to overcome that problem I used "SliverToBoxAdapter". So that it can contain login button as separate children in it as compared to adding it to list tile.

Reply:
Please just add two widgets to the `children`. They don't need to be `ListTile`s.